README.md

```diff
@@ -105,19 +106,21 @@ export ECR_CROMWELL_TAG=<some-value>
 export ECR_NEXTFLOW_ACCOUNT_ID=<some-value>
 export ECR_NEXTFLOW_REGION=<some-value>
 export ECR_NEXTFLOW_TAG=<some-value>
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell and Nextflow engine respectively
```

Comment:
Not really part of the PR but can you please add the MiniWDL values above as well?

Reply:
No problem